src/components/atm.button/button.styled.jsx

```diff
@@ -0,0 +1,22 @@
+import styled from 'styled-components';
+import {FontFamily, FontWeight, FontSize, Color, Border, Padding, ButtonHeight} from '../constants'
+
+const buttonColor = {
+    primary: Color.Primary,
+    secondary: Color.Secondary
+}
+
+export const ButtonStyled = styled.button`
```

Comment:
Talvez n tratou o estado de hover pq eh soh mobile? se for o caso, tem alguns props especificos do mobile, como -webkit-tap-highlight-color, se quiser 🙂